modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,40 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected databases. 
+{company} has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration (ZDM) Proxy
+* Cassandra Data Migrator
+* DSBulk Loader
+ 
+With these tools, you can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+Your clusters are kept in sync at all times by a dual-write logic configuration.
+You can roll back at any point for complete peace of mind.
+
+== Migration process and tools
+
+{company} can migrate all data, however critical, with or without downtime from one application to our database. 
+When migrating data, you begin with an application connected to a cluster. 
+You run data and connect your own users to this application.
+Reads and writes are moved (as necessary) from the application to the cluster.
+After completing the migration, the data runs from your cluster to our database.
+Our data migration process is about the same, but with several options to make the transition smooth. 
+//The migration process differs slightly with each of our migration tools. 
+
+* *ZDM Proxy*: You can continue to run your current application and migrate data to the Origin database without any downtime. 
+It's a process of moving your current data from your application to the {company}'s Origin cluster seamlessly without stopping any connection to and from any cluster or application. 
+The proxy helps to manage the activity in transition.
+Your applications connect directly to the proxy instead of pointing to the database.
+NOTE: We recommend using three ZDM Proxies for this option. 
+* *Cassandra Data Migrator (CDM)*: This option migrates and validates data between Cassandra-compatible clusters. 
+It supports migration at scale and ensures accuracy and efficiency during the process.
+CDM allows for migration with zero downtime when used with the ZDM Proxy, and it can also be used for migrations with acceptable downtime. 
+* *DSBulk Loader*: This option is loading and unloading CSV/JSON data and counting table rows. 
+ DSBulk Loader does not inherently support zero downtime migrations like CDM when used with ZDM Proxy.
```

Comment:
DSBulk also allows for migration with zero downtime when used with the ZDM Proxy. It does not support data validation / reconciliation.

Reply:
I'd make a slight adjustment to what Faizal had proposed above,
> It does not support data validation / reconciliation

to below,
> It does not support data validation / reconciliation other than validating the final table counts